e2e/test/provisioning/ProvisioningServiceIndividualEnrollmentTests.cs

```diff
@@ -0,0 +1,431 @@
+
+            BulkEnrollmentOperationResult createBulkEnrollmentResult = await provisioningServiceClient
+                .IndividualEnrollments
+                .RunBulkOperationAsync(BulkOperationMode.Create, createIndividualEnrollmentList)
+                .ConfigureAwait(false);
+
+            createBulkEnrollmentResult.IsSuccessful.Should().BeTrue();
+
+            // update two enrollments in bulk
+            IndividualEnrollment retrievedEnrollment1 = await provisioningServiceClient.IndividualEnrollments.GetAsync(registrationId1).ConfigureAwait(false);
+            retrievedEnrollment1.Capabilities = new InitialTwinCapabilities
+            {
+                IsIotEdge = true,
+            };
+            IndividualEnrollment retrievedEnrollment2 = await provisioningServiceClient.IndividualEnrollments.GetAsync(registrationId2).ConfigureAwait(false);
+            retrievedEnrollment2.Capabilities = new InitialTwinCapabilities
+            {
+                IsIotEdge = true,
+            };
+            var updateIndividualEnrollmentList = new List<IndividualEnrollment> { retrievedEnrollment1, retrievedEnrollment2 };
+
+            BulkEnrollmentOperationResult updateBulkEnrollmentResult = await provisioningServiceClient
+                .IndividualEnrollments
+                .RunBulkOperationAsync(BulkOperationMode.Update, updateIndividualEnrollmentList)
+                .ConfigureAwait(false);
+
+            updateBulkEnrollmentResult.IsSuccessful.Should().BeTrue();
+
+            // delete two enrollments in bulk
+            retrievedEnrollment1 = await provisioningServiceClient.IndividualEnrollments.GetAsync(registrationId1).ConfigureAwait(false);
+            retrievedEnrollment2 = await provisioningServiceClient.IndividualEnrollments.GetAsync(registrationId2).ConfigureAwait(false);
+            var deleteIndividualEnrollmentList = new List<IndividualEnrollment> { retrievedEnrollment1, retrievedEnrollment2 };
+
+            BulkEnrollmentOperationResult deleteBulkEnrollmentResult = await provisioningServiceClient
+                .IndividualEnrollments
+                .RunBulkOperationAsync(BulkOperationMode.Delete, deleteIndividualEnrollmentList)
+                .ConfigureAwait(false);
+
+            deleteBulkEnrollmentResult.IsSuccessful.Should().BeTrue();
+        }
+
+        private static async Task ProvisioningServiceClient_GetIndividualEnrollmentAttestation(AttestationMechanismType attestationType)
+        {
+            using var provisioningServiceClient = new ProvisioningServiceClient(TestConfiguration.Provisioning.ConnectionString);
+            string registrationId = AttestationTypeToString(attestationType) + "-" + Guid.NewGuid();
+
+            IndividualEnrollment individualEnrollment = null;
+
+            try
+            {
+                individualEnrollment = await CreateIndividualEnrollmentAsync(
```

Comment:
nit: tabbing looks a bit off
```suggestion
                individualEnrollment = await CreateIndividualEnrollmentAsync(
                    provisioningServiceClient,
                    registrationId,
                    attestationType,
                    null,
                    null,
                    AllocationPolicy.Static,
                    null).ConfigureAwait(false);
```

Reply:
Nope, that is intended. Too many parameters so each get their own line, and we have ConfigureAwait after, which we'd prefer on the next line. So, the vars get an extra tab.

If this was more fluent in style, you'd do another newline on the method, so something like:

```csharp
var foo = MyObj
    .MethodAsync(
        param1,
        param2)
    .ConfigureAwait(false);
```

Reply:
Thank you for clarifying!